dev/com.ibm.ws.webcontainer.security.app/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -78,6 +78,9 @@ useAuthenticationDataForUnprotectedResource.desc=Specifies whether authenticatio
 loginFormURL=Login form URL 
 loginFormURL.desc=Specifies the global URL of a form login page including the root context. The form login page must be part of the WAR file. If a form login application does not specify the form login page in the web.xml file, it uses the global form login URL.
 
+loginErrorURL=Login error URL 
+loginErrorURL.desc=Specifies the global URL of a form login error page including the root context. The form login error page must be part of a WAR file. If an application using form-login authentication does not specify either the form login page or the login error page in the auth-method element of the web.xml file, it uses the global login error URL.
```

Comment:
Suggested changes:
The login error URL specifies the global URL of a form login error page that includes the root context. The form login error page must be part of a WAR file. An application uses the global login error URL if the application uses form login authentication and does not specify either the form login page or the login error page in the auth-method element of the web.xml file.

Reply:
Changed as suggested.